### ISSUE TITLE
Feature: Allow player to request a hint

### ISSUE THREAD

**Issue by bgri** [bgri/m100LE#48]
Just capturing this as an item so we don't lose it.

         Perhaps at some point we can consider a way to ask the player if they want to see what the answer was...

_Originally posted by @hackerb9 in https://github.com/bgri/m100LE/issues/10#issuecomment-1312570753_
      
Could be something simple like pressing the [?] key. Right side screen will then show current correct char/positions, with an incorrect/unguessed character revealed.

**Comment by hackerb9** [comment]
I like this idea, although we'll have to do #28 first to fit it in. (My Tandy 200 has a max of 24K per bank, so it is already having trouble running the version with the comments. Even TEENY.CO can't be loaded in RAM.) 

Also, it may be good to reserve <kbd>?</kbd> for showing brief help instructions, if ever there is enough space for showing a couple sentences. Perhaps another key can be found for hints (<kbd>!</kbd>)?

**Comment by bgri** [comment]
Good call. <kbd>?<kbd> for brief help is a great idea.<kbd>!<kbd> it is.

Once we reduce the code size.
